pkgs/development/python-modules/pytest-xdist/default.nix

```
@@ -25,9 +26,11 @@ buildPythonPackage rec {
   buildInputs = [
     pytest
   ];
-  checkInputs = [ pytestCheckHook filelock ];
+  checkInputs = [ pytestCheckHook filelock pexpect ];
   propagatedBuildInputs = [ execnet pytest-forked psutil ];
 
+  pytestFlagsArray = [ "-n $NIX_BUILD_CORES" "--forked" ]; # pytest can already use xdist at this point
```

I do prefer longopts in non-interactive usage and ideally one argument per line for better diffs.

```suggestion
  pytestFlagsArray = [
    # pytest can already use xdist at this point
    "--numprocesses $NIX_BUILD_CORES"
    "--forked"
  ];
```

The `--forked` argument is apparently on the way out: https://github.com/pytest-dev/pytest-forked